qcodes/utils/validators.py

```diff
@@ -45,12 +45,20 @@ class Validator:
     base class for all value validators
     each should have its own constructor, and override:
 
+    __init__: here a private attribute, _valid_values, should be set.
```

Comment:
I would prefer a different name like `_representative_values` or `_mock_values`, because `_valid_values` implies that it might be a complete list. (Which of course does not make sense on second thought)

Reply:
Since it's a private attribute, it can have any old name. But of course it should have the name of the corresponding property, and I was also a bit in doubt about a good name. `valid_values` kind of breaks tab completion for `validate`, but I don't really see an obviously better alternative. `allowed_values`, `representative_values`, ...?

Note that it is actually a complete list for `Enum` validators.